src/runtime/cache.ts

```diff
@@ -135,7 +136,11 @@ export function defineCachedEventHandler (handler: CompatibilityEventHandler, op
       if (opts.magAge) {
         cacheControl.push(`s-maxage=${opts.magAge}`)
       }
-      cacheControl.push('stale-while-revalidate')
+	  if (opts.staleMaxAge) {
+		  cacheControl.push(`stale-while-revalidate=${opts.staleMaxAge}`)
```

Comment:
I was thinking since we are keeping stale value up to `maxAge` anyway does it make sense to default it to `opts.maxAge`?

Reply:
Mostly you don't use the same values. I think leaving multiple options open to a developer is better.

Reply:
I see, however, this is the actual behavior of cachify function right now. We only revalidate until maxAge reaches not before.

Reply:
Yes but that is not the point behind `swr`

Reply:
`stale-while-revalidate` header alone tells browser always revalidate no?

Reply:
Yes, but with the options you can define when it should revalidate and not only when the response expired.

Reply:
I know but we _already know_ that data is not going to be invalidated until maxAge. But really we can improve default generated value later ;)

Reply:
You have convinced me.